packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -27,6 +28,30 @@ function resolveAbsoluteRuntime(moduleName: string, dirname: string) {
   }
 }
 
+function resolveRuntimeVersion(
+  moduleName: string,
+  modulePath: string,
+  dirname: string,
+) {
+  try {
+    const runtimeDir = path.dirname(
+      resolve.sync(`${modulePath}/package.json`, {
+        basedir: dirname,
+      }),
+    );
+    const pkgFilename = path.resolve(runtimeDir, "../../../package.json");
```

Comment:
Hmm, I'm not sure this is what I'd recommend for verifying this. Could we consider looking for the `package.json` in a parent directory starting from `dirname`? It seems like treating the root as relative to the plugin usage location might be safer than using the location of the plugin, since that could be in an unknown place.